app/controllers/admin/partners_controller.rb

```diff
@@ -0,0 +1,59 @@
+module Admin
+  class PartnersController < AdminController
+    include Search
+
+    def index
+      @partners = Partner
+
+      if params[:q].present?
+        # Returns an ActiveRecord::Relation of publishers for pagination
+        search_query = "%#{remove_prefix_if_necessary(params[:q])}%"
+        @partners = Partner.where(search_sql, *[search_query] * 7).distinct
+      end
+
+      @partners = @partners.suspended if params[:suspended].present?
+      @partners = @partners.where(created_by: current_user) if params[:created_by_me].present?
+
+      @partners = @partners.order(created_at: :desc).paginate(page: params[:page])
+    end
+
+    def new
+      @partner = Partner.new
+    end
+
+    def create
+      # Find any existing publishers so we don't create duplicate entries
+      @partner = partner
+
+      if @partner.persisted? && (@partner.partner? || @partner.admin?)
+        flash.now[:alert] = "Email is already a partner"
+        render "new"
```

Comment:
Would you use the symbol?

Reply:
Yup sure thing - https://github.com/brave-intl/publishers/pull/1354/commits/9f2060a32ca89c8ea4c8dfaa5c7a8a9e7a39c60b